docs/general/profiles.md

```diff
@@ -39,12 +39,14 @@ They may be used in any Profiles record they apply to.
 | `beam` | [Erlang](https://en.wikipedia.org/wiki/BEAM_(Erlang_virtual_machine)) | ![Development](https://img.shields.io/badge/-development-blue) |
 | `cpython` | [Python](https://wikipedia.org/wiki/Python_(programming_language)) | ![Development](https://img.shields.io/badge/-development-blue) |
 | `dotnet` | [.NET](https://wikipedia.org/wiki/.NET) | ![Development](https://img.shields.io/badge/-development-blue) |
+| `go` | [Go](https://wikipedia.org/wiki/Go_(programming_language)), | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
Should we clarify whether this should be used for symbolized frames only (otherwise 'native'), or could it span both symbolized and unsymbolized?

Reply:
If the producer can distinguish in more detail the frame type, `go` must be preferred over `native`. Regardless whether the frame is symbolized or not.